scripts/pr_test/pr_super_lint/npm_ci.sh

```diff
@@ -2,6 +2,6 @@
 
 bash "${GITHUB_WORKSPACE}/scripts/npm_ci.sh"
 echo "PYTHONPATH=/github/workspace/:/github/workflow/.venv/lib/python$(echo 'import sys; print(".".join(map(str, sys.version_info[0:2])))' | python)/site-packages" >> "${GITHUB_ENV}"
-action="$(yq '.jobs.pr-super-lint.steps[-1].uses' .github/workflows/pr-test.yml)"
+action="$(yq '.jobs.pr-super-lint.steps[-1].uses | line_comment' .github/workflows/pr-test.yml)"
 PATH="$(docker run --rm --entrypoint '' "ghcr.io/${action//\/slim@/:slim-}" /bin/sh -c 'echo $PATH')"
```

Comment:
https://github.com/dev-hato/hato-bot/actions/runs/7595032465/job/20687157847?pr=3606

```
Unable to find image 'ghcr.io/v5.7.2:latest' locally
docker: Error response from daemon: Head "https://ghcr.io/v2/v5.7.2/manifests/latest": name invalid.
See 'docker run --help'.
```

```suggestion
PATH="$(docker run --rm --entrypoint '' "ghcr.io/super-linter/super-linter:slim-${action}" /bin/sh -c 'echo $PATH')"
```

こうするのがいいか。

Reply:
[`97424e8` (#3606)](https://github.com/dev-hato/hato-bot/pull/3606/commits/97424e8b34d653e2983bc47a3effbe2ed8bc8f89)
ここで入れました